### PR TITLE
Add user stories tooltips to the Telemetry pages

### PR DESCRIPTION
**Pull request by asankov**

### CODE REVIEW

File: tcms/telemetry/templates/telemetry/testing/breakdown.html [hunk 1]
```diff
@@ -7,7 +7,10 @@
 {% block contents %}
 
     <div class="container-fluid container-cards-pf">
-
+        <h1 style="margin-top: 0">
+          <span>{% trans "Testing Breakdown" %}</span>
+          <button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="right" title="{% trans 'As a tester and/or manager I want to see a breakdown of tests so that I have an idea how the existing tests for the product are distributed' %}" style="margin-left: 5px">?</button>
```
Comment (atodorov) [inline tcms/telemetry/templates/telemetry/testing/breakdown.html]: Drop the h1 and button tags, use
`<span class="fa pficon-help"`

- no inline CSS
- we don't need titles
- move the help icon to the right of the screen
- use descriptions from website

File: tcms/telemetry/static/telemetry/css/testing/telemetry.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.help-tooltip {
+  float: right; 
+  font-size: 2.5rem; 
+  padding-right: 2rem;
+}
```
Comment (atodorov) [inline tcms/telemetry/static/telemetry/css/testing/telemetry.css]: new line missing

File: tcms/telemetry/static/telemetry/css/testing/telemetry.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.help-tooltip {
+  float: right; 
+  font-size: 2.5rem;
```
Comment (atodorov) [inline tcms/telemetry/static/telemetry/css/testing/telemetry.css]: no need to specify font-size

File: tcms/telemetry/static/telemetry/css/testing/telemetry.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.help-tooltip {
+  float: right;
```
Comment (atodorov) [inline tcms/telemetry/static/telemetry/css/testing/telemetry.css]: trailing white space

File: tcms/telemetry/templates/telemetry/testing/breakdown.html [hunk 5]
```diff
@@ -27,6 +26,7 @@
                         <option value="">----------</option>
                     </select>
                 </div>
+                <span class="fa pficon-help help-tooltip" data-toggle="tooltip" data-placement="left" title="{% trans 'As a tester and/or manager I want to see a breakdown of tests so that I have an idea how the existing tests for the product are distributed' %}"></span>
```
Comment (atodorov) [inline tcms/telemetry/templates/telemetry/testing/breakdown.html]: Place this at the top of the form like the rest b/c consistency and b/c it doesn't look nice once you resize the screen.

File: tcms/telemetry/templates/telemetry/testing/execution-trends.html [hunk 6]
```diff
@@ -2,13 +2,15 @@
...
+{% block title %}{% trans "Execution trends" %}{% endblock %}
 
 {% block contents %}
 
     <div class="container-fluid container-cards-pf">
-
-        <form class="form-horizontal">
+      <form class="form-horizontal">
+            <div class="form-group">
+              <div class="fa pficon-help help-tooltip" data-toggle="tooltip" data-placement="left" title="{% trans 'As a test manager I want to see information if Product builds are getting better or worse' %}"></div>
```
Comment (atodorov) [inline tcms/telemetry/templates/telemetry/testing/execution-trends.html]: use descriptions from home page, everywhere.

File: tcms/telemetry/static/telemetry/css/testing/telemetry.css (hunk 3 above)
Comment (asankov) [reply tcms/telemetry/static/telemetry/css/testing/telemetry.css]: okay, but the the icon is really small

**Pull request merged**